RHIDP-9010: AEM-related changes (#1426)
redhat-developer/red-hat-developers-documentation-rhdh: modules/customizing-the-appearance/proc-customize-rhdh-page-theme.adoc

            shape: none
# ...
----

> `light`:: The theme mode, for example, `light` or `dark`
> `default`:: The `yaml` header for the default page theme configuration
> `backgroundColor`:: The color of the page header background, for example, `#ffffff` or `white`
> `fontColor`:: The color of the text in the page header, for example, `#000000` or `black`
> `shape`:: The pattern on the page header, for example, `wave`, `round`, or `none`
> `apis::` The `yaml` header for a specific page theme configuration, for example, `apis`, `home`

Reviewer: ```suggestion
`light`:: Enter the theme mode, such as `light` or `dark`.
`default`:: Enter the default page theme configuration
`backgroundColor`::: Enter the page header background color, such as `#ffffff` or `white`.
`fontColor`::: Enter the page header text color, such as `#000000` or `black`.
`shape`::: Enter the page header pattern, such as `wave`, `round`, or `none`.
`apis::` Enter the page id to configure, such as `apis` or `home`.
```
